src/task/CreatePeerConnectionTask.ts

```diff
@@ -24,7 +29,12 @@ export default class CreatePeerConnectionTask extends BaseTask implements Remova
   ];
   private removeVideoTrackEventListeners: { [trackId: string]: (() => void)[] } = {};
 
-  constructor(private context: AudioVideoControllerState) {
+  static readonly removeHandlerIntervalMs: number = 10000;
```

Comment:
We may not be perfectly consistent about this, but in general we use ALL_CAPS_CASE when defining constants to make it clear that they are constants, e.g.

https://github.com/aws/amazon-chime-sdk-js/blob/master/src/activespeakerdetector/DefaultActiveSpeakerDetector.ts#L16

Reply:
done